data/compatible_products.yml

```diff
@@ -27,10 +27,10 @@ sets:
 
     - name: Open Liberty
       vendor: IBM Corporation
-      version: 19.0.0.6
+      version: 20.0.0.3
       image: '/images/compatible_products/ibm_open_liberty.png'
       download: 'https://openliberty.io/downloads/'
-      compatibility: 'https://github.com/jakartaee/jakarta.ee/issues/439'
+      compatibility: 'https://github.com/jakartaee/jakarta.ee/issues/855'
```

Comment:
I thought these links were supposed to be for the Certification Requests against the Platform repo.  Looking at our Compatibility Page (https://jakarta.ee/compatibility/), we're not consistent.  Some products list the Certification Request (ie. Payara), while others list the Compatible Software Request Issue (ie. Open Liberty).  I think it should be the Certification Request, which in this case would be https://github.com/eclipse-ee4j/jakartaee-platform/issues/227 for the Platform.